dts/bindings/watchdog/st,stm32-window-watchdog.yaml

```diff
@@ -0,0 +1,29 @@
+#
+# Copyright (c) 2019 Centaur Analytics, Inc
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+title: STMicroelectronics STM32 system window watchdog driver
+
+description: >
+    This is a representation of the STM32 system window watchdog
+
+inherits:
+    !include base.yaml
+
+properties:
+    compatible:
+      constraint: "st,stm32-window-watchdog"
+
+    reg:
+      category: required
+
+    label:
+      category: required
+
+    interrupts:
+      category: required
+
```

Comment:
extract from H7/MP1 user manual: " The early wakeup interrupt (EWI) can be used if specific safety operations or data logging must be performed before the actual reset is generated.
So should be "optional" instead of "required"

Reply:
Done